.yarnrc.yml

```diff
@@ -5,9 +5,6 @@ enableGlobalCache: true
 preferAggregateCacheInfo: true
 
 logFilters:
-  - pattern: 'fsevents@*/* The platform * is incompatible with this module, link skipped.'
```

Comment:
I think we still need this, unfortunately:

```sh
➜  wp-calypso git:(chore/update-yarn) yarn --version
3.1.0
➜  wp-calypso git:(chore/update-yarn) yarn
➤ YN0000: ┌ Resolution step
➤ YN0000: │ Some peer dependencies are incorrectly met; run yarn explain peer-requirements <hash> for details, where <hash> is the six-letter p-prefixed code
➤ YN0000: └ Completed in 0s 484ms
➤ YN0000: ┌ Fetch step
➤ YN0013: │ 3464 packages were already cached, 6 had to be fetched
➤ YN0000: └ Completed in 7s 213ms
➤ YN0000: ┌ Link step
➤ YN0076: │ fsevents@patch:fsevents@npm%3A2.3.2#~builtin<compat/fsevents>::version=2.3.2&hash=18f3a7 The linux-x64 architecture is incompatible with this module, link skipped.
➤ YN0076: │ fsevents@patch:fsevents@npm%3A1.2.13#~builtin<compat/fsevents>::version=1.2.13&hash=18f3a7 The linux-x64 architecture is incompatible with this module, link skipped.
➤ YN0076: │ fsevents@patch:fsevents@npm%3A2.1.2#~builtin<compat/fsevents>::version=2.1.2&hash=18f3a7 The linux-x64 architecture is incompatible with this module, link skipped.
➤ YN0008: │ wp-calypso@workspace:. must be rebuilt because its dependency tree changed
➤ YN0000: └ Completed in 10s 998ms
➤ YN0000: Done with warnings in 19s 534ms

```